packages/demo/src/demo-building-blocs/Guidelines.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import {Components} from 'react-markdown';
+import {Tooltip} from 'react-vapor';
+
+const MarkdownFiles = new Map();
+
+const importAll = (context: __WebpackModuleApi.RequireContext) => {
+    context.keys().forEach((key) => {
+        const [, formattedKey] = /.*\/(\w+)\.md$/.exec(key);
+        MarkdownFiles.set(formattedKey, context(key).default);
+    });
+};
+
+importAll(require.context('!raw-loader!../../docs', false, /\.md$/));
+
+export const Guidelines = {
+    exists: (fileName: string): boolean => MarkdownFiles.has(fileName),
+    get: (fileName: string): string => MarkdownFiles.get(fileName),
+};
+
+const Table: Components['table'] = ({children}) => React.createElement('table', {className: 'table my2'}, children);
+
+const OrderedList: Components['ol'] = ({children}) => React.createElement('ol', {className: 'list-decimal'}, children);
+
+const UnorderedList: Components['ul'] = ({children}) => React.createElement('ul', {className: 'list-disc'}, children);
+
+const Emphasis: Components['em'] = ({children}) =>
+    React.createElement('em', {className: 'body-m-book-italic'}, children);
+
+const Strong: Components['strong'] = ({children}) => React.createElement('strong', {className: 'body-m'}, children);
+
+const Link: Components['a'] = ({title, children, ...props}) =>
+    React.createElement(
+        Tooltip,
+        {
+            title: title || '',
+        },
+        React.createElement(
+            'a',
+            {
+                ...props,
+                // if the link is relative don't open in a new tab
+                target: props.href && props.href.match(/^http/) ? '_blank' : '',
+                rel: 'noopener noreferrer',
+                className: 'link',
+            },
+            children
+        )
+    );
```

Comment:
Would it work with tsx?

```suggestion
    <Tooltip title={title}>
      <a {...props} target={props.href && props.href.match(/^http/) ? '_blank' : ''} rel="noopener noreferrer" className="link">
        {children}
      </a>
  </Tooltip>
```
 I have the same question for all the other elements

Reply:
Good point, tried it and it works 👍  Updating all the overrides